src/App.tsx

```diff
@@ -51,8 +61,10 @@ function App() {
   const handleThemeToggle = () => {
     if (currentThemePalette === "dark") {
       changeCurrentThemePalette("light");
+      localStorage.setItem("savedTheme", "light");
```

Comment:
dont hardcode keynames for local storage variables here, instead keep them in a new config file

Reply:
done